### DISCUSSION TITLE
wixtoolset.heat 4.0.0-rc.3 heat.exe virus detected W32/Exploit.gen (Panda Endpoint Protection)

### DISCUSSION THREAD

**Discussion by maniglia** [wixtoolset/issues#7251] [Questions] [answered]
Antivirus Panda Endpoint Protection (WatchGuard EPP) detect the heat.exe file as a virus and deletes it.

_.nuget\packages\wixtoolset.heat\4.0.0-rc.3\tools\net472\x86\heat.exe_

The RC.2 version works fine.

Regards

**Comment by jchoover** [comment, marked as answer]
https://www.virustotal.com/gui/url/bc26c089432856a00661ea3896ef41eec72ac4b0d2733cbbe8c918026f41e69e?nocache=1

Sounds like a Panda issue to me.

**Reply by maniglia** [reply]
I agree that the exe does not contain any virus and it is definitely a false positive detected by WatchGuard antivirus, I will try to report [here](https://www.watchguard.com/wgrd-support/security-portal/report-false-positive).

**Comment by maniglia** [comment]
[Here](https://www.trendmicro.com/it_it/research/23/a/batloader-malware-abuses-legitimate-tools-uses-obfuscated-javasc.html) you can find a detailed explanation of why the antivirus detects WiX tools as viruses.
Basically WiX tools are misused by malicious actors (bad guys), and that's why the antivirus intervenes.
